microshift_configuring/microshift_low_latency/microshift-low-latency.adoc

```diff
@@ -0,0 +1,77 @@
+= Configuring low latency
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-low-latency
+
+toc::[]
+
+You can configure and tune low latency capabilities to improve application performance on edge devices.
+
+include::modules/microshift-low-latency-concept.adoc[leveloffset=+1]
+
+//additional resources for the low latency concept
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/scalability_and_performance/low-latency-tuning#cnf-understanding-low-latency_cnf-understanding-low-latency[About low latency] (OpenShift Container Platform documentation)
+
+include::modules/microshift-install-rpms-low-latency.adoc[leveloffset=+1]
+
+include::modules/microshift-low-latency-config-yaml.adoc[leveloffset=+1]
+
+//additional resources for the config.yaml
+[role="_additional-resources"]
+.Additional resources
+* xref:../../microshift_configuring/microshift-using-config-tools.adoc#microshift-config-yaml_microshift-configuring[Using a YAML configuration file]
+* link:https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration[KubeletConfiguration reference] (Kubernetes upstream documentation)
+* link:https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/#create-the-config-file[Create the config file] (Kubernetes upstream documentation)
+
+//RHEL TuneD
+include::modules/microshift-low-latency-tuned-conc.adoc[leveloffset=+1]
+
+//microshift-baseline is the name of the profile
+//microshift-baseline-variables.conf is the file for user tweaks
+include::modules/microshift-low-latency-tuned-profile.adoc[leveloffset=+2]
+
+//additional resources for tuned profiles
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/monitoring_and_managing_system_status_and_performance/getting-started-with-tuned_monitoring-and-managing-system-status-and-performance[Getting started with TuneD] (RHEL documentation)
+* link:https://www.redhat.com/sysadmin/linux-tuned-tuning-profiles[How to manage tuning profiles in Linux] (Red Hat blog)
+
+//microshift-baseline is the name of the profile and used for automatic activation settings
+include::modules/microshift-low-latency-tuned-auto.adoc[leveloffset=+2]
+
+//RHEL real-time kernel
+include::modules/microshift-low-latency-kernelrt-conc.adoc[leveloffset=+1]
+
+include::modules/microshift-low-latency-install-kernelrt-rpm.adoc[leveloffset=+2]
+
+include::modules/microshift-low-latency-install-kernelrt-rhel-edge.adoc[leveloffset=+2]
+
+//from rpm-ostree install guide
+include::modules/microshift-creating-ostree-iso.adoc[leveloffset=+3]
+
+include::modules/microshift-add-blueprint-build-iso.adoc[leveloffset=+3]
+
+include::modules/microshift-download-iso-prep-for-use.adoc[leveloffset=+3]
+
+include::modules/microshift-provisioning-ostree.adoc[leveloffset=+3]
```

Comment:
I think these create a distraction when reading the docs, also because they do not appear in the TOC.
Can we consider referencing those instead of including them?